scripts/dotnet-cli-build/PublishTargets.cs

```diff
@@ -95,6 +95,12 @@ public static BuildTargetResult FinalizeBuild(BuildTargetContext c)
                         AzurePublisherTool.CopyBlob(blob.Replace("/dotnet/", ""), targetUrl);
                     }
 
+                    // Copy the latest installer files
+                    CopyBlobs($"{Channel}/Installers/{CliNuGetVersion}/", $"{Channel}/Installers/Latest/");
```

Comment:
Can you also refactor the copying of the the CLI Binaries folder into something like -
`CopyBlobs($"{Channel}/Binaries/{CliNuGetVersion}/", $"{Channel}/Binaries/Latest/");` ?

So that the code looks uniform and we dont do anything special for CLI binaries.